packages/concerto-core/test/data/parser/classdeclaration.badidentifierprefix.cto

```diff
@@ -11,16 +11,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
-/*eslint-disable no-unused-vars*/
-/**
- * This is a transaction processor function
- *
- * @throws {Error}
- * @exception {Error}
- */
-function onTestTransactionLulz() {
-    console.log('onTestTransaction');
+namespace com.testing
+
+asset Asset identified by id {
+  o String id
+  o String validate // This is ok
```

Comment:
@jeromesimeon This file is used in unit tests to prove that previously reserved values can now be used.